.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  root: true,
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+  ],
+  parserOptions: {
+    project: './tsconfig.json',
+  },
+  ignorePatterns: ['dist/**/*'],
+  rules: {
+    '@typescript-eslint/no-explicit-any': 'error',
+    '@typescript-eslint/no-unsafe-argument': 'off',
+    '@typescript-eslint/no-unsafe-member-access': 'off',
+    '@typescript-eslint/no-unsafe-assignment': 'off',
+    '@typescript-eslint/no-unsafe-call': 'off',
+    '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
In addition, none of these suppressed rules should be necessary, now that `@types/node-fetch` exist

Reply:
@wkurniawan07 I checked again and it seems fine to remove all the suppressed rules now.